### PR TITLE
Update biopython to 1.69

### PR DESCRIPTION
**Pull request by souravsingh**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bgruening** [comment]
This was fast @souravsingh! :)

**Comment by souravsingh** [comment]
I got the message from the biopython mailing list just a few minutes ago regarding the update. So I went ahead with the PR.

File: recipes/biopython/meta.yaml [hunk 1]
```diff
@@ -9,12 +9,12 @@ about:
...
-  fn: biopython-1.68.tar.gz
-  url: https://pypi.python.org/packages/72/6c/e1e13b9df73f9c2539b67d12bc22be6b19779230cadbed04c24f3f3e5ef4/biopython-1.68.tar.gz
-  md5: 078e915185485a5327937029b7577ddc
+  fn: biopython-1.69.tar.gz
+  url: https://pypi.python.org/pypi/biopython/1.69
+  md5: 18ad299569eea79febf4641cce840db0
 
 build:
-  number: 0
+  number: 1
```
Comment (bgruening) [inline recipes/biopython/meta.yaml]: New versions should start with 0 again.

**Comment by bgruening** [comment]
This is great @souravsingh!

**Comment by cbrueffer** [comment]
I get the same CI error in #4380 , seems to be something wrong on the Travis side?

**Pull request closed without merging**